spec/models/package_spec.rb

```diff
@@ -35,9 +37,13 @@
     it { is_expected.to validate_presence_of(:package_type_id) }
     it { is_expected.to_not allow_value(-1).for(:quantity) }
     it { is_expected.to_not allow_value(-1).for(:received_quantity) }
+    it { is_expected.to_not allow_value(-1).for(:weight) }
+    it { is_expected.to_not allow_value(-1).for(:pieces) }
```

Comment:
These two test cases can be removed, as those are covered at line 47

Reply:
sure @swatijadhav , actually that's a typo, i wanted to check for value 0, will fix it.